src/Services/UserService.php

```diff
@@ -142,6 +142,16 @@ public function create($data, $userId = null)
 
     /**
      * @param string $userId
+     *
+     * @return User
+     */
+    public function totpReset($userId)
```

Comment:
Perhaps `resetTopt()` would be better, name of method should describe intention, unlike URL section should be noun, name of resource.